tests/functional/docs/test_help_output.py

```diff
@@ -177,6 +177,10 @@ def test_elb_help_output(self):
         # .. _`:
         self.assert_not_contains('.. _`:')
 
+    def test_shorthand_flattens_list_of_single_member_structures(self):
```

Comment:
Are there other functional tests that cover the remaining cases?  I think it's worth having functional tests at least one per service, just to have that extra safety net.

If there's existing assert_params_for_cmd functional tests for the other commands/params then I think we're good with functional tests.